authserver/api/static/css/style.css

```diff
@@ -54,52 +83,50 @@ label.form-check-label {
 
 .alert-danger {
   color: #f00000;
-  background-color: #ffb3b3;
-  border-color: #ff7a7a;
+  background-color: rgba(255, 0, 0, 0.15);
+  border: solid 0.5px #ff0000;
   font-weight: 500;
 }
 
 /* Buttons */
-.btn-primary {
-  background-color: #0097a7;
-  border: 3px solid #0097a7;
-  border-radius: 0;
-}
-
-.btn-primary:hover,
-.btn-primary:disabled {
-  background-color: #66c1ca;
-  border: 3px solid #66c1ca;
-}
-
-.btn-primary:disabled {
-  cursor: not-allowed;
+.btn {
+  border-radius: 4px;
+  font-weight: 500;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  letter-spacing: 0.2px;
+  font-stretch: normal;
+  font-style: normal;
+  box-sizing: border-box;
+  white-space: nowrap;
+  border: none;
 }
 
-.btn-secondary {
-  background-color: #333333;
-  border: 3px solid #0097a7;
-  border-radius: 0;
+.btn-primary {
+  background-color: var(--blue);
 }
 
-.btn-lg {
-  font-size: 18px;
+.btn-primary:hover {
+  background-color: var(--blue);
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25) !important;
 }
 
-.btn-link {
-  display: block;
-  padding: 0;
-  border: none;
-  color: inherit;
+.btn-primary:active {
+  background-color: var(--blue) !important;
 }
 
-.btn-link:hover {
-  text-decoration: none;
-  color: #0097a7;
+.btn-primary:disabled {
+  background-color: var(--light-blue);
+  cursor: not-allowed;
```

Comment:
disabled buttons should have `pointer-events: none`, this will turn the cursor to normal and stop any click events on it

Reply:
good catch!

https://github.com/brighthive/authserver/blob/e0dd4b2c3ca15c36f48e1af8562fdf0b772e14b3/authserver/api/static/css/style.css#L105